cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractAssemblyOperator.java

```diff
@@ -321,4 +322,51 @@ public boolean onlyMetricsSettingChanged(ConfigMap current, ConfigMap desired) {
         }
         return onlyMetricsSettingChanged && diff.size() == 1;
     }
+
+    private class NamespacedResourceName    {
```

Comment:
This can be `static`, no?

It also might be useful elsewhere, so maybe we should make it top level, wdyt?